edge-modules/Relayer/Program.cs

```diff
@@ -72,5 +79,17 @@ static async Task<MessageResponse> ProcessAndSendMessageAsync(Message message, o
 
             return MessageResponse.Completed;
         }
+
+        static async Task ReportStatus(TestResultCoordinatorClient trcClient, string moduleId, string message)
+        {
+            try
+            {
+                await trcClient.ReportResultAsync(new TestOperationResult { Source = moduleId, Result = message, CreatedAt = DateTime.UtcNow, Type = moduleId });
```

Comment:
as discussed, type should be 'messages', 'twin', 'directmethod' etc.

Reply:
you need to put "trackingId", "batchId" and "sequenceNumber" with delimiter into TestOperationResult.Result.